docs/3.0rc/get-started/quickstart.mdx

```diff
@@ -129,101 +142,120 @@ Prefect automatically tracks the state of the flow run and logs the output in th
 14:28:33.255 | INFO    | Flow run 'energetic-panther' - Finished in state Completed('All states completed.')
 ```
 
-## Deploy on a remote infrastructure
+## Create a work pool
 
-Running a flow locally is a good start, but a remote destination is used for production. Tell Prefect where to run the workflow by creating a [work pool](/3.0rc/deploy/work-pools/control-runs/).
+Running a flow locally is a good start, but you should use a remote destination for production flows.
+A [work pool](/3.0rc/deploy/work-pools/control-runs/) is the most common way to do this.
 
-Prefect Cloud can run the flow code using a [Prefect Managed work pool](/3.0rc/deploy/serve-flows/index). You can create a work pool in the UI or from the CLI.
+<Tabs>
+  <Tab title="Cloud">
```

Comment:
Do we want suggest a Process type work pool for Cloud and OSS? Then they would be the same setup and both expose folks to workers. Docker also introduces come more install and complexity that we may want to avoid for the quickstart.